ctgan/data_transformer.py

```diff
@@ -108,10 +111,11 @@ def _transform_continuous(self, column_transform_info, raw_column_data):
             component_porb_t = component_probs[i] + 1e-6
             component_porb_t = component_porb_t / component_porb_t.sum()
             selected_component[i] = np.random.choice(
-                np.arange(num_components), p=component_porb_t)
+                np.arange(num_components), p=component_porb_t
```

Comment:
Even though the plugin does not complain, I would split this in two lines:

```python3
selected_component[i] = np.random.choice(
    np.arange(num_components),
    p=component_porb_t)
)
```

Otherwise, I think that everything would just fit in a single line.